src/stories/Quiz/Hooks/index.tsx

```diff
@@ -106,6 +107,7 @@ export default function HooksTemplate(args) {
       if (currentQuestion.isOpenQuestion) {
         return (
           <div className='cio-quiz'>
+            <style>.cio-quiz {convertPrimaryColorsToString(primaryColorStyles)}</style>
```

Comment:
`primaryColorStyles` wasn't being used in the hooks example. But this isn't a friendly customer interface so we still need to handle this in a separate PR maybe